### DISCUSSION TITLE
Problem with Docker and blackfire

### DISCUSSION THREAD

**Discussion by sergey-voloshin02** [symfony/symfony#45723] [Symfony Book: The Fast Track] [answered]
I cant start my server with docker:

12/03 09:48:17 DBG Reading agent configuration file "/dev/null"

  The server ID parameter is not set; please run 'blackfire agent:config' to configure it.

**Comment by lyrixx** [comment, marked as answer]
Hello, I invite you to contact [blackfire](https://support.blackfire.io/en/) since this is not related to Symfony.
